lib/components/viewers/styled.js

```diff
@@ -1,18 +1,58 @@
-import styled from 'styled-components'
+import styled, { keyframes } from 'styled-components'
 
 /** Route Details */
-export const Container = styled.div`
-  background-color: ${(props) =>
-    props.full ? props.backgroundColor || '#ddd' : 'inherit'};
-  color: ${(props) => (props.full ? props.textColor : 'inherit')};
-  height: 100%;
-  overflow-y: hidden;
-`
 
 export const RouteNameContainer = styled.div`
   padding: 8px;
   background-color: inherit;
 `
+
+const backgroundColorTransition = keyframes`
+    from {
+      background-color: white;
+    }
+`
+const fade = keyframes`
+    0% { opacity: 0%; }
+    100 { opacity: 100%; }
+`
+const fadeDelay = keyframes`
+0% { opacity: 0%; }
+50% { opacity: 0% }
+100 { opacity: 100%; }
+`
+
+export const StyledRouteViewerContainer = styled.div`
+  animation: ${(props) => props.patternView && backgroundColorTransition} 150ms
+      linear,
+    ${(props) => (props.fade ? fade : 'none')} 100ms reverse linear forwards;
+  background-color: ${(props) =>
+    props.routeColor ? props.routeColor : '#fff'};
+  color: ${(props) => (props.textColor ? props.textColor : 'inherit')};
```

Comment:
This and the other props can be written as:
```suggestion
  color: ${(props) => (props.textColor || 'inherit')};
```